events/class_tests/possible_participant_tests.py

```diff
@@ -30,12 +30,12 @@ def get_event():
 
 @pytest.fixture
 def get_possible_event_participants():
-    return [PossibleParticipant.objects.get(id=f'{id}') for id in range(3, 7)]
+    return [PossibleParticipant.objects.get(id=f'{id}') for id in range(3, 8)]
```

Comment:
8 or 7 ... the difference, and which one is correct, may be clear now, but use of literals explicitly may become hard to understand (and maintain) later on, so I think it is clearer to use named constants (or enumerations where relevant)

Reply:
Done 😄